.gitlab/ci/on-push.yml

```diff
@@ -273,33 +284,37 @@ mpv2-prepare-testing-bucket:
 
 xsoar_server_6_5:
   extends: .test_content_on_xsoar_server_instances_base
-  #  No need to trigger in case of release branch
+  #  No need to trigger in case of release branch or docker update branches (non-nightly packs)
   rules:
+    - !reference [.filter-non-nightly-docker-updates-rule, rules]
     - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH !~ /^[0-9]{2}\.[0-9]{1,2}\.[0-9]$/'
   variables:
     INSTANCE_ROLE: "Server 6.5"
 
 xsoar_server_6_6:
   extends: .test_content_on_xsoar_server_instances_base
-  #  No need to trigger in case of release branch
+  #  No need to trigger in case of release branch or docker update branches (non-nightly packs)
   rules:
+    - !reference [.filter-non-nightly-docker-updates-rule, rules]
     - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/ && $CI_COMMIT_BRANCH !~ /^[0-9]{2}\.[0-9]{1,2}\.[0-9]$/'
```

Comment:
Can we remove all the conditions on release branches? we are not performing this for some tine now

Reply:
I can remove those rules and test if we don't use them.
Are you sure they are not in use?

Reply:
99%. Let's remove and re-run tests.

Reply:
Will do!